pkg/protocol/protobuf/errors/errors.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.9
+// 	protoc        v5.26.1
```

Comment:
We're going to have a lot of churn if different people are using different protoc versions to generate these files. Not a big deal, but it does make it harder to tell at a glance if a .pb.go file was modified in a substantive way without a corresponding update to the proto source file.

Things to consider for future PRs:
 * Could we generate the pb.go files automatically at merge time?
 * Short of that, could we regenerate the pb.go files and check for discrepancies as a pre-merge check?
 * If we do the latter, we should include a Dockerfile for building the proto files using a fixed protoc version.

Reply:
good callout, created https://github.com/teslamotors/vehicle-command/issues/307

Reply:
Updated to generate with the proper protoc version. Looks like there are still a few unnecessary changes but much better.